README.md

```diff
@@ -102,16 +102,6 @@ const App = () => (
 <style type="text/css" id="ssr-styles">${sheetsRegistry.toString()}</style>
 ```
 
-## Examples
-
-To run the example react app:
-
-```sh
-> cd examples/react-app
-> npm i
-> npm start
-```
-
```

Comment:
Happy to hear if this example app provides value that I'm not aware of, in case it might be worth keeping/maintaining.

Reply:
I does not add any value that I'm aware of @RohanM. I cannot see it being more valuable than Storybook, so I think we could :fire: it.